internal/logger/logger_test.go

```diff
@@ -103,44 +123,63 @@ var _ = Describe("Logger", func() {
 	})
 })
 
-func basicInstance(name, guid string) ccapi.ServiceInstance {
-	return fullInstance(name, guid, false, "fake-op-type", "fake-op-state")
+func createFailedInstance() ccapi.ServiceInstance {
+	return ccapi.ServiceInstance{
+		Name:             "create-failed-instance",
+		GUID:             "create-failed-instance-guid",
+		UpgradeAvailable: true,
+		LastOperation: ccapi.LastOperation{
+			Type:  "create",
+			State: "failed",
+		},
+	}
+}
```

Comment:
praise: thank you very much. It is easier to understand the obtained data when something fails